timelapse/recorder.py

```diff
@@ -29,44 +29,44 @@ def __init__(self, output_dir, interval=4):
         self._stopped = Event()
 
         # Set config options
-        self.output_dir = output_dir
-        self.interval = interval
-        self.format = "png"
+        self.output_dir: str = output_dir
+        self.interval: int = interval
+        self.format: str = "png"
 
         # Number of screenshots taken
-        self.screenshot_counter = 0
+        self.screenshot_counter: int = 0
         print("Recorder started")
 
-    def join(self, timeout=None):
+    def join(self, timeout=None) -> None:
         """ Stop recording """
         self._stop.set()
         self._stopped.wait()
         print("Recorder stopped. Total recording time: " +
               self.get_recording_time() + ".")
         Process.join(self, timeout=timeout)
 
-    def run(self):
+    def run(self) -> None:
         """ Periodically take a screenshots of the screen """
         while not self._stop.is_set():
             self.screenshot()
             self._stop.wait(self.interval)
 
         self._stopped.set()
 
-    def get_recording_time(self):
+    def get_recording_time(self) ->str:
         return str(self.screenshot_counter * self.interval) + " seconds"
 
-    def get_filename(self):
+    def get_filename(self) ->str:
         """ Call screencapture for mac screenshot """
-        filename = os.path.join(
+        filename: str = os.path.join(
             self.output_dir, f"{self.screenshot_counter}.{self.format}")
         return filename
 
-    def screenshot(self):
+    def screenshot(self) -> None:
         """ This method uses Mac OSX screencapture from the commandline """
-        filename = self.get_filename()
+        filename: str = self.get_filename()
         subprocess.run(
             ['screencapture', '-S', '-o', '-x', '-D',
                 str(get_screen_with_mouse_index() + 1), '-t', self.format, filename],
-            check=True)
-        self.screenshot_counter += 1
+            check: bool =True)
```

Comment:
This is a parameter that we pass to `subprocess.run`. `check=True` is equivalent to writing `True` in this case. We just use a named parameter here because it's more readable for 👨 👩. Guess we have to remove this type hint to make it syntactically correct.

Reply:
Done 👍 bb76c7c